### ISSUE TITLE
Fail to publish in Lumen

### ISSUE THREAD

**Issue by BosNaufal** [Folkloreatelier/laravel-graphql#62]
Hey, It's Awesome Package~ :grin: :+1: 
But I just wondering why I got error when trying to publish the configuration of graphql package in my Lumen. The error look like this:

```bash
$ php artisan graphql:publish
  PHP Fatal error:  Call to undefined method Folklore\GraphQL\LumenServiceProvider::command() in /var/www/proyek/goodmovie/vendor/folklore/graphql/src/Folklore/GraphQL/LumenServiceProvider.php on line 50

  [Symfony\Component\Debug\Exception\FatalErrorException]
  Call to undefined method Folklore\GraphQL\LumenServiceProvider::command()
```

Did I make any mistake?

**Comment by dmongeau** [comment]
Thanks!

There was a mistake in the service provider, sorry about that. The method should be `->commands()` and not `->command()`. It is fixed in 1.0.3

**Comment by BosNaufal** [comment]
Hey @dmongeau,

Thanks for replying me :grin: but there's a new problem. It seems $app['route'] in lumen is not defined or route->pattern is not defined in Lumen :confused: check it out~

Error:
```bash
$ php artisan graphql:publish
  PHP Fatal error:  Call to undefined method Laravel\Lumen\Application::pattern() in /var/www/proyek/goodmovie/vendor/folklore/graphql/src/Folklore/GraphQL/ServiceProvider.php on line 48

  [Symfony\Component\Debug\Exception\FatalErrorException]
  Call to undefined method Laravel\Lumen\Application::pattern()
```

Check this out, maybe it can help [https://github.com/laravel/lumen-framework/issues/113](https://github.com/laravel/lumen-framework/issues/113)

**Comment by dmongeau** [comment]
Ok it should be fixed in v1.0.4, let me know if you have any other issue with lumen

**Comment by BosNaufal** [comment]
@dmongeau,
Maybe it's not the problem of package. But I can install the 1.0.4 version and the master version is pointing on v1.0.3. Am I doing it wrong?

```bash
$ composer update
Loading composer repositories with package information

Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package folklore/graphql 1.0.4 exists as folklore/graphql[dev-develop, dev-master, v0.1.0, v0.1.1, v0.2.0, v
0.2.1, v0.2.2, v0.2.3, v0.2.4, v0.2.5, v0.3.0, v0.3.1, v0.3.2, v0.3.3, v0.4.0, v0.4.1, v0.4.2, v0.4.3, v0.4.4, v0.4.5, v0.4.6, v
0.4.7, v0.4.8, v0.4.9, v1.0.0, v1.0.1, v1.0.2, v1.0.3, v1.x-dev] but these are rejected by your constraint.
```
